src/components/Scrollback.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  BackgroundProps,
+  LayoutProps,
+  PositionProps,
+  TypographyProps,
+  background,
+  layout,
+  position,
+  typography,
+} from "styled-system";
+import React, { Fragment, useCallback } from "react";
+
+import styled from "styled-components";
+import theme from "./theme";
+
+const Button = styled.button<
+  PositionProps & TypographyProps & BackgroundProps & LayoutProps
+>`
+  ${position};
+  ${typography};
+  ${background};
+  ${layout};
+  border: none;
+`;
+
+interface ButtonProps {
+  bottom: number;
+  left: number;
+}
+
+const Scrollback: React.FC<ButtonProps> = props => {
+  const { bottom, left } = props;
+  const scrollToTop = useCallback(() => {
+    if (window) {
+      window.scrollTo({ top: 0, behavior: "smooth" });
+    }
+  }, []);
+  return (
+    <Fragment>
```

Comment:
i dont think you need the `Fragment`, you only really need it when there are 2 jsx elements side by side

eg. if Scrollback was:

```
return (
  <Fragment>
    <div></div>
    <Button />
  </Fragment>
)
```